src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleExtractToSpecificPath.cs

```diff
@@ -57,6 +57,28 @@ private void Bundle_Extraction_To_Specific_Path_Succeeds()
             extractDir.Should().NotHaveFiles(BundleHelper.GetFilesNeverExtracted(fixture));
         }
 
+        [InlineData("./foo")]
+        [InlineData("../foo")]
+        [InlineData("foo")]
+        [InlineData("foo/bar")]
+        [Theory]
+        private void Bundle_Extraction_To_Relative_Path_Succeeds (string relativePath)
+        {
+            var fixture = sharedTestState.TestFixture.Copy();
+            var singleFile = BundleHelper.BundleApp(fixture, BundleOptions.None);
+
+            // Run the bundled app (extract files to <path>)
+            Command.Create(singleFile)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .EnvironmentVariable(BundleHelper.DotnetBundleExtractBaseEnvVariable, relativePath)
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdOutContaining("Hello World");
```

Comment:
Nit: If it's not hard, you may also validate that it created the right folder and ran from there.